src/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2020 EPAM Systems, Inc. All Rights Reserved. All information contained herein is, and remains the
+ * property of EPAM Systems, Inc. and/or its suppliers and is protected by international intellectual
+ * property law. Dissemination of this information or reproduction of this material is strictly forbidden,
+ * unless prior written permission is obtained from EPAM Systems, Inc
+ */
+
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { MovieItem } from '../MovieItem'
+import styles from './MovieList.scss'
+
+export const MovieList = ({ res }) => {
+  const onHover = (setState) => setState(true)
```

Comment:
All these methods related to MovieItem
Why MovieListShould know about them?
onModal should be moved to the modal component